index.html

```diff
@@ -174,15 +174,15 @@ <h3 class="styleh3">CONTACT US</h3>
     <p class="division_sections"></p>
     <footer class="footer_class">
       <div class="logos">
-        <img src="img/Vector.png" alt="GitHub">
-        <img src="img/dos.png" alt="LinkedIn">
-        <img src="img/instagram.png" alt="Instagram">
-        <img src="img/cuatro.png" alt="Medium">
+        <img src=".img/Vector.png" alt="GitHub">
+        <img src=".img/dos.png" alt="LinkedIn">
+        <img src=".img/instagram.png" alt="Instagram">
+        <img src=".img/cuatro.png" alt="Medium">
       </div>
       <p class="emails">businesswitharmando@gmail.com</p>
     </footer>
     <p class="division_footer"></p>
   </div>
-  <script src="/javascript.js"></script>
```

Comment:
### Your Design
![navbar (copy)](https://user-images.githubusercontent.com/93039359/179650915-3f03122e-6ea8-415a-b080-28ca9c92940d.png)

- [ ] You can see in the image that there are some green boxes and red boxes. Green boxes show the correct position for the contents which are in the red boxes. Please make them on the correct position to follow the Figma template :blush: